package.json

```diff
@@ -9,6 +9,7 @@
     "react": "yarn workspace @aws-amplify/ui-react",
     "react-auth": "yarn workspace @aws-amplify/ui-react-auth",
     "react-core": "yarn workspace @aws-amplify/ui-react-core",
+    "react-core-auth": "yarn workspace @aws-amplify/ui-react-core-auth",
```

Comment:
Would love to see more details in the PR description so that anyone reading these changes can grok why separate package is needed or what the future intent is regarding `react-core-auth` and `react-core`. 
Usually when the PR description is sparse I look at other documentation or tests to understand purpose but the readme just says it's an internal pkg.